src/harness/util.py

```diff
@@ -337,7 +343,71 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
   request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
   request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
 
-
+     
+def computePropagationAntennaModel(request):
+    reliability_level = request['reliabilityLevel']
+    if reliability_level not in [-1, 0.05, 0.95]:
+        response = 400
```

Comment:
Please raise exceptions rather than passing back magic values. We should take advantage of the features of Python rather than restricting ourselves to C-style solutions.

Reply:
There are two ways we can implement this.
1) Is to return a http 400 error when incorrect values are sent to the SAS. In which case,m it should not be considered a failed test if the SAS returns an error for a incorrect parameters.
2) The system throw an exception when an input is wrong.

Reply:
The thing is that this isn't actually an HTTP error, it's just a magic value that's being returned from this function. That makes it difficult to debug and counterintuitive to people who are expecting Python-style code.

I think you can throw an exception when the input is wrong. Then the call to this function within the test case would look something like:

```
try:
  # call ref prop model
except InvalidArgumentError as e:
  # log invalid params for debugging by the tester
  # increment a "failed configs" counter so that the % of passed configs is correctly calculated at the end of the test
  # skip this config

# call the SAS UUT's prop model function (if we've made it this far, the ref model has already validated the input)
# comparison of SAS UUT, ref model

percentage_of_passed_tests = num_passed_tests / (num_total_tests - num_failed_configs)
```